static/src/stylesheets/module/content-garnett/_gallery.head.scss

```diff
@@ -1,65 +1,229 @@
-.gallery__main-content {
-    .content__meta-container {
-        position: relative;
-        margin-left: 0;
-        min-height: auto;
-        margin-bottom: $gs-baseline;
+.immersive-header-container {
+    background-color: $garnett-neutral-1;
+    overflow-x: hidden;
+}
+
+.immersive-header-container--photo-essay,
+.js-adblock-sticky {
```

Comment:
`.js` classes shouldn't be used for styling. They are only for use in javaScript.